src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -1003,4 +1013,8 @@
   </xsl:attribute-set>
   -->
 
+  <!-- Provide enough space for two digit labels (needed in bibliography). -->
```

Comment:
It can be fixed by adding the following:

```xml
<xsl:attribute-set name="orderedlist.label.properties">
  <xsl:attribute name="text-align">right</xsl:attribute>
</xsl:attribute-set>
```